exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/CreateTableTests.kt

```diff
@@ -237,29 +215,45 @@ class CreateTableTests : DatabaseTestsBase() {
             assertEquals(1, ddlId1.size)
             assertEquals("ALTER TABLE $tableProperName ADD ${Person.id1.descriptionDdl(false)}", ddlId1.first())
 
-            assertEquals(1, ddlId2.size)
-            assertEquals(
-                "ALTER TABLE $tableProperName ADD ${Person.id2.descriptionDdl(false)}, ADD CONSTRAINT $pkConstraintName PRIMARY KEY ($id1ProperName, $id2ProperName)",
-                ddlId2.first()
-            )
+            when (testDb) {
+                in TestDB.ALL_H2, TestDB.ORACLE -> {
+                    assertEquals(2, ddlId2.size)
+                    assertEquals("ALTER TABLE $tableProperName ADD $id2ProperName ${Person.id2.columnType.sqlType()}", ddlId2.first())
+                    assertEquals("ALTER TABLE $tableProperName ADD CONSTRAINT $pkConstraintName PRIMARY KEY ($id1ProperName, $id2ProperName)", Person.id2.ddl.last())
+                }
+                else -> {
+                    assertEquals(1, ddlId2.size)
+                    assertEquals(
+                        "ALTER TABLE $tableProperName ADD ${Person.id2.descriptionDdl(false)}, " +
+                            "ADD CONSTRAINT $pkConstraintName PRIMARY KEY ($id1ProperName, $id2ProperName)",
+                        ddlId2.first()
+                    )
+                }
+            }
         }
     }
 
     @Test
-    fun addOneColumnPrimaryKeyToTableNotH2Test() {
-        withTables(excludeSettings = TestDB.ALL_H2, tables = arrayOf(Book)) {
+    fun addOneColumnPrimaryKeyToTableTest() {
+        withTables(excludeSettings = TestDB.ALL_H2, tables = arrayOf(Book)) { testDb ->
```

Comment:
Could H2 also be included? I added it to the same branch as Oracle and the test passed.